project/plugins.sbt

```diff
@@ -1,9 +1,10 @@
-addSbtPlugin("com.github.sbt" % "sbt-site"      % "1.7.0")
-addSbtPlugin("com.github.sbt" % "sbt-release"   % "1.4.0")
-addSbtPlugin("com.github.sbt" % "sbt-ghpages"   % "0.8.0")
-addSbtPlugin("org.scoverage"  % "sbt-scoverage" % "2.2.1")
-addSbtPlugin("org.scoverage"  % "sbt-coveralls" % "1.3.14")
-addSbtPlugin("com.github.sbt" % "sbt-pgp"       % "2.3.0")
+addSbtPlugin("com.github.sbt" % "sbt-site"       % "1.7.0")
+addSbtPlugin("com.github.sbt" % "sbt-release"    % "1.4.0")
```

Comment:
I think we can remove the sbt-release and the sbt-pgp plugin 
(sbt-pgp is automatically brought in by the sbt-ci-release) ?